### ISSUE TITLE
Update to 1.08.03+ breaks on Lenovo Smart Clock2

### ISSUE THREAD

**Issue by Breeches6786** [marcokreeft87/room-card#349]
**Which version of Room Card?**
1.08.03 & above

**Describe the bug**
Updating to 1.08.03 or above breaks the ability to load room-card on the Lenovo Smart Clock2. The cards fail to load and say "Custom element doesn't exist: room-card"

I believe it's the same issue that another user had with mod-card here: https://community.home-assistant.io/t/please-stop-forcing-us-to-buy-brand-new-dashboard-hardware-and-make-gui-stuff-work-on-older-webview-again/655827

**Comment by marcokreeft87** [comment]
I don't have that device so I can't debug. Don't know what's the problem. But it seems like home assistant can't find the code. You have to reload the frontend cache after installing custom cards. 

That's all I can do for you. Sorry. The problem isn't the card.

**Comment by Breeches6786** [comment]
This card worked fine for 2 years previously, and rolling back to 1.08.02 restores functionality. It's not a frontend cache issue. I believe the issue is a change to the webview requirements, and the device's webview cannot be updated (thanks Lenovo)

If it can't be updated to work, that's fine. I'll just stay on 1.08.02 or find another card. Thank you for the work you've done on it. It's appreciated.

**Comment by marcokreeft87** [comment]
Yeah sorry, I wish I could help. But tbf, there is not really new stuff in 1.08.03 besides some update for the new lovelace
